### DISCUSSION TITLE
AWS: ERROR celery.worker.consumer.consumer consumer: Cannot connect to amqp://guest:**@127.0.0.1:5672//: [Errno 111] Connection refused.

### DISCUSSION THREAD

**Discussion by btodorce** [saleor/saleor#8719] [Deployment]
Deployed the backend stack at aws, but worker keeps restarting with this error
ERROR celery.worker.consumer.consumer consumer: Cannot connect to amqp://guest:**@127.0.0.1:5672//: [Errno 111] Connection refused.

**Comment by karolkielecki** [comment]
@btodorce Maybe this comment regarding Celery will be helpful: https://github.com/saleor/saleor/discussions/8707#discussioncomment-1743308. If you still have some issues with it, please let me know.
